Makefile

```diff
@@ -44,6 +46,9 @@ start-dockerdemo:
 clean-dockerdemo:
 	scripts/dockerhost/cleanup-dockerhosts
 
+ssh:
```

Comment:
Yes; I looked into that, but the only good way to do that without running a bunch of other stuff is to eliminate the dependency. This is mostly help for dev.